examples/weight-init/README.md

```diff
@@ -0,0 +1,93 @@
+# Weight Initialization
+
+Table of Contents
+=================
+* [Introduction](#introduction)
+* [Selecting a weight initialization strategy](#selecting-a-weight-initialization-strategy)
+* [Experiments with LeNet on MNIST](#experiments-with-lenet-on-mnist)
+
+## Introduction
+
+Weight initialization can critically affect the speed at which a neural network is able to learn. Under certain circumstances,
+a poor initialization of weights can prevent a neural network from learning anything.
+
+In this series of examples we will see how a flavor of the canonical LeNet neural network is able to learn from the MNIST
+dataset, under various weight initialization schemes.
+
+## Selecting a weight initialization strategy
+
+### Under Caffe
+
+Caffe makes it straightforward to select one of the various supported weight initialization methods. In the `prototxt` definition
+of the network, each layer can specify how to initialize weights through a `weight_filler` field . For example, in order to set the
+initial weights to a constant `0.2`:
+
+```
+weight_filler {
+      type: "constant"
+      value: 0.2
+    }
+```
+
+The initialization of the bias terms can also be specified through a `bias_filler` field, although this tends to have a lesser impact
+on learning so we will focus on the initialization of weights in the following examples.
+
+### Under Torch7
+
+Most layers make it straightforward to initialize weights randomly from a uniform distribution over `[-std, std]` by calling their `reset(std)` method. However the determination
```

Comment:
Can you post a little bit of code here to show how to actually do this for a layer in your network (like you did above for Caffe)? I would hope to be able to read this guide and then know enough to go in and edit the default Torch AlexNet model to change the weight initialization.